docker/docker-entrypoint.sh

```diff
@@ -122,27 +122,37 @@ install_languages() {
 	if [ ${#langs[@]} -eq 0 ]; then
 		return
 	fi
-	apt-get update
 
+	# Build list of packages to install
 	for lang in "${langs[@]}"; do
 		pkg="tesseract-ocr-$lang"
 
 		if dpkg --status "$pkg" &>/dev/null; then
 			echo "Package $pkg already installed!"
 			continue
+		else
+			to_install+=($pkg)
```

Comment:
```suggestion
			to_install+=("$pkg")
```

Probably won't matter here, but for consistency: This prevents word splitting/globbing.

Reply:
Done.